src/components/Molecules/Banner/Banner.js

```diff
@@ -3,14 +3,17 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
 import spacing from '../../../theme/shared/spacing';
+import { screen } from '../../../theme/shared/size';
 
 const Wrapper = styled.div`
   background: ${({ theme, backgroundColor }) => theme.color(backgroundColor)};
 `;
 
 const Container = styled.div`
+  text-align: center;
   padding: calc(${spacing('m')} * 2) ${spacing('m')};
   margin: 0 auto;
+  max-width: ${screen.large};
 
   @media ${({ theme }) => theme.breakpoint('small')} {
     padding: ${spacing('xl')} 5%;
```

Comment:
Does it need to be "5%"? Maybe just add a `spacing` or as it will be centred max-width could be smaller.